apps/web/src/examples/MountedViews.tsx

```diff
@@ -0,0 +1,94 @@
+import Basemap from '@arcgis/core/Basemap';
+import {
+  CalciteBlock,
+  CalciteLabel,
+  CalciteOption,
+  CalciteSelect,
+} from '@esri/calcite-components-react';
+import {
+  ArcMapView,
+  ArcSceneView,
+  MountedViewsProvider,
+  useArcState,
+  useMountedViews,
+} from 'arcgis-react';
+
+export default function Example() {
+  return (
+    <MountedViewsProvider>
+      <div style={{ height: '60%', display: 'flex' }}>
+        {/* Map View */}
+        <ArcMapView
+          id="my-map-view"
+          style={{ flex: 1 }}
+          map={{ basemap: 'streets' }}
+          zoom={3}
+          center={[-100.4593, 36.9014]}
+        />
+
+        {/* Scene View */}
+        <ArcSceneView
+          id="my-scene-view"
+          style={{ flex: 1 }}
+          map={{ basemap: 'streets' }}
+          camera={{
+            tilt: 45,
+            heading: 0,
+            position: { x: 0, y: 40, z: 1000000 },
+          }}
+        />
+      </div>
+
+      {/**
+       * Basemap Picker - outside of components directly rendering
+       * a Map or Scene
+       */}
+      <BaseMapPickList />
+    </MountedViewsProvider>
+  );
+}
+
+function BaseMapPickList() {
+  const views = useMountedViews();
+  if (!views) return null;
+
+  return (
+    <CalciteBlock heading="Basemap options" open>
+      {Object.entries(views).map(([id, view]) => {
+        if (!view) return null;
+
+        return (
+          <BasemapPicker key={id} label={`Set Basemap for ${id}`} view={view} />
+        );
+      })}
+    </CalciteBlock>
+  );
+}
```

Comment:
```suggestion
function BaseMapPickList() {
  const { mySceneView, myMapView } = useMountedViews();

  return (
    <CalciteBlock heading="Basemap options" open>
      {myMapView && (
        <BasemapPicker label="Set Basemap for mapView" view={myMapView} />
      )}
      {mySceneView && (
        <BasemapPicker label="Set Basemap for sceneView" view={mySceneView} />
      )}
      <ArcBasemapGallery view={mySceneView} />
    </CalciteBlock>
  );
}
```

i feel like this might be a better way to convey to the users that you can get a view directly from their ID that you define on the ArcMapView/ArcSceneView.